crates/sui-indexer-alt/migrations/2024-10-21-003426_tx_indices/up.sql

```diff
@@ -0,0 +1,39 @@
+CREATE TABLE IF NOT EXISTS tx_affected_addresses (
+    tx_sequence_number          BIGINT       NOT NULL,
```

Comment:
What's our latest thought on NOT NULL? I think all the columns here are truly non nullable so I went with NOT NULL.

Reply:
I think `NOT NULL` is fine for key columns (which are all the columns in the filter tables). In my PRs, I omitted the `NOT NULL` constraints on the columns that are part of the primary key, because being part of the primary key implies `NOT NULL`, but I'm happy to go back and change that -- I think not having the explicit `NOT NULL` was confusing in its own way.

Reply:
Yes I think having explicit `NOT NULL` even for primary keys would be better.